### PR TITLE
Homebuying Controller Tests

### PR DESCRIPTION
**Pull request by gatorjuice**
homebuying controller 100% coverage

### CODE REVIEW

File: spec/controllers/homebuyings_controller_spec.rb [hunk 1]
```diff
@@ -1,5 +1,160 @@
...
+      end
+    end
+
+    describe "it attempts to save the homebuying instance" do
+      context "saves successfully" do
+        before :each do
+          @this_client = create(:client)
+          sign_in @this_client
+        end
+        it "creates a new programemployee" do
```
Comment (heydanhey) [inline spec/controllers/homebuyings_controller_spec.rb]: Didn't we decide to remove the ProgramEmployee stuff? I don't think it's being used anymore. What do you think?

File: spec/controllers/homebuyings_controller_spec.rb [hunk 2]
```diff
@@ -1,5 +1,160 @@
...
+        some_homebuying = create(:homebuying)
+        sign_in some_user
+        get :edit, id: some_homebuying
+        expect(assigns(:foreclosure)).to eq(some_homebuying)
+      end
+    end
+    it "renders the #edit template" do
+      get :edit, id: create(:homebuying)
+      expect(response).to render_template :edit
+    end
```
Comment (heydanhey) [inline spec/controllers/homebuyings_controller_spec.rb]: Cool, I see you added the `edit.html.erb` file to get this test to pass. But it's never used in the app?

**Comment by heydanhey** [comment]
+1, Looks great! I'm basically using this PR as a template for testing the rentals controller. I left two comments - but they are more of questions we can discuss at standup.

File: spec/controllers/homebuyings_controller_spec.rb (hunk 1 above)
Comment (gatorjuice) [inline spec/controllers/homebuyings_controller_spec.rb]: lets discuss at stand. I took it out for now

File: spec/controllers/homebuyings_controller_spec.rb (hunk 2 above)
Comment (gatorjuice) [inline spec/controllers/homebuyings_controller_spec.rb]: so the controller is gonna try to do something at the end. I just wrote the test for the case given. We can also discuss what we want edit to do

**Pull request merged**